pkgs/applications/misc/rofi-rbw/default.nix

```diff
@@ -21,6 +21,10 @@ buildPythonApplication rec {
 
   pythonImportsCheck = [ "rofi_rbw" ];
 
+  preFixup = ''
+    makeWrapperArs+=(--prefix PATH : ${lib.makeBinPath [ rbw ]})
```

Comment:
```suggestion
    makeWrapperArgs+=(--prefix PATH : ${lib.makeBinPath [ rbw ]})
```

Reply:
Argh! I did test this, but have rbw in my PATH now, so I didn't notice. Sorry for that.